src/permissionGroups/components/MembersErrorDialog/MembersErrorDialog.tsx

```diff
@@ -0,0 +1,49 @@
+import DialogContentText from "@material-ui/core/DialogContentText";
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import ActionDialog from "@saleor/components/ActionDialog";
+import { ConfirmButtonTransitionState } from "@saleor/components/ConfirmButton";
+
+export interface MembersErrorDialogProps {
+  confirmButtonState: ConfirmButtonTransitionState;
+  open: boolean;
+  onConfirm: () => void;
+  onClose: () => void;
+}
+
+const MembersErrorDialog: React.FC<MembersErrorDialogProps> = ({
+  confirmButtonState,
+  onClose,
+  onConfirm,
+  open
+}) => {
+  const intl = useIntl();
+
+  return (
+    <ActionDialog
+      open={open}
+      confirmButtonState={confirmButtonState}
+      onClose={onClose}
+      onConfirm={onConfirm}
+      title={intl.formatMessage({
+        defaultMessage: "Unassign users",
+        description: "dialog title"
+      })}
+      variant="default"
+    >
+      <DialogContentText>
+        <FormattedMessage
+          defaultMessage="You are not able to unassign {counter,plural,one{this member} other{{displayQuantity} members}} from group. Solve this problem to continue with request."
+          description="dialog content"
+          values={{
+            counter: 1,
+            displayQuantity: <strong>{1}</strong>
```

Comment:
I believe that this should be some variable. Also `intl` won't translate it - whole message should be wrapped in plural condition.